Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 
-gem 'support', git: 'git@github.com:sinatra/mustermann.git'
+gem 'support', git: 'https://github.com/sinatra/mustermann.git'
```

Comment:
I had a infinite error running `budnle` for this gem so I need to change it to use HTTPS instead of git. Let me know if you find it redundant

Reply:
Why are we trying to take a dependency on a git repo instead of a released version here?

Reply:
Notably this causes an incompatibility running `rake`.

```
/Users/dblock/.rvm/rubies/ruby-2.6.2/lib/ruby/2.6.0/bundler/runtime.rb:319:in `check_for_activated_spec!': You have already activated mustermann 1.0.3, but your Gemfile requires mustermann 1.1.0. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)
```

Reply:
Oh it's a gem within a gem? Confusing :) I'll fix.